install.sh

```diff
@@ -34,4 +34,14 @@ cp -R "$BATS_ROOT"/libexec/* "$PREFIX"/libexec
 cp "$BATS_ROOT"/man/bats.1 "$PREFIX"/share/man/man1
 cp "$BATS_ROOT"/man/bats.7 "$PREFIX"/share/man/man7
 
+# fix broken symbolic link file
+if [ ! -L "$PREFIX"/bin/bats ]; then
+    rm -f "$PREFIX"/bin/bats
+    ln -s "$PREFIX"/libexec/bats "$PREFIX"/bin/bats
```

Comment:
I just tried this on Git for Windows, and for some reason `ln -s` doesn't want to cooperate:

```bash
$ ./install.sh ../usr
ln: failed to create symbolic link '../usr/bin/bats': No such file or directory
```

Since the "link" is just a copy on this platform anyway, perhaps we should just use `cp`?

Reply:
It seems some issue on NTFS
ref: [1](https://cygwin.com/ml/cygwin/2015-05/msg00122.html) 
I got the solutionfrom [2](https://stackoverflow.com/questions/4175264/bash-retrieve-absolute-path-given-relative)
assign absolute path instead of relative path.
It works with `git-bash.exe`